Traceback (most recent call last):
  File "main.py", line 51, in <module>
    main()
  File "main.py", line 48, in main
    train_loop(siamese, dataloader, ContrastiveLoss(), optim.Adam(siamese.parameters(), lr=0.0005), EPOCHS, RCS, SR, THRESHOLD_VC, N, vowels, OFFSET)
  File "/storage1/zhang.ning/Active/Oliver/Medical-SoK/icd_coding_multirescnn/Voice-Similarity/train_siamese.py", line 37, in train_loop
    output1, output2 = network(layer_1_tensor, layer_2_tensor)
  File "/opt/conda/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/storage1/zhang.ning/Active/Oliver/Medical-SoK/icd_coding_multirescnn/Voice-Similarity/siamese.py", line 50, in forward
    output1 = self.forward_once(intput1)
  File "/storage1/zhang.ning/Active/Oliver/Medical-SoK/icd_coding_multirescnn/Voice-Similarity/siamese.py", line 47, in forward_once
    x = self.fc(x)
  File "/opt/conda/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/opt/conda/lib/python3.8/site-packages/torch/nn/modules/container.py", line 204, in forward
    input = module(input)
  File "/opt/conda/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/opt/conda/lib/python3.8/site-packages/torch/nn/modules/linear.py", line 114, in forward
    return F.linear(input, self.weight, self.bias)
RuntimeError: mat1 and mat2 shapes cannot be multiplied (1x5088 and 5120x1280)
